deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -252,12 +252,13 @@ def update_watchdir(self, watchdir_id):
                 # Skip directories
                 continue
             else:
-                ext = os.path.splitext(filename)[1]
+                ext = os.path.splitext(filename)[1].lower()
                 if ext == ".torrent":
                     magnet = False
                 elif ext == ".magnet":
                     magnet = True
                 else:
+                    log.warning("File checked for auto-loading is invalid: %s", filename)
```

Comment:
I'm not sure this is needed. If a user uses a directory that contains other files in it then a lot of log messages will be generated.